lib/types/signature.ex

```diff
@@ -13,5 +13,6 @@ defmodule Kadena.Types.Signature do
 
   @impl true
   def new(sig) when is_binary(sig), do: %__MODULE__{sig: sig}
+  def new(sig) when is_nil(sig), do: []
```

Comment:
@EdwinGuayacan why is this required? why nil is not an invalid value?

Reply:
because they might get a `SignCommand` with the sig value as nil
![image](https://user-images.githubusercontent.com/80716239/209860399-fc7e6cde-5feb-4cf2-8b82-34efca196a20.png)

Reply:
@EdwinGuayacan So I think this is just a workaround to solve that specific use case. Use a pattern matting when sig is nil and skip the creation of Signature!